Dockerfile.ci-rp

```diff
@@ -70,6 +70,8 @@ FROM ${REGISTRY}/ubi8/ubi-minimal AS final
 LABEL aro-final=true
 RUN microdnf update && microdnf clean all
 COPY --from=builder /app/aro /app/e2e.test /usr/local/bin/
+# Copy coverage report to final image
+COPY --from=builder /app/cover.out /app/cover.out
```

Comment:
We won't want to do this. Instead, let's [extract](https://unix.stackexchange.com/questions/331645/extract-file-from-docker-image) the `cover.out` from our tagged rp-build image in the Makefile at the end of `ci-rp`